baybe/utils/basic.py

```diff
@@ -67,6 +68,34 @@ def set_random_seed(seed: int):
     np.random.seed(seed)
 
 
+@contextlib.contextmanager
+def temporary_seed(seed: int):  # noqa: DOC402, DOC404
```

Comment:
shouldnt this and the above function be in `utils.numeric` or even something like `utils.random` ?

Reply:
can do